packages/polymath-issuer/src/pages/sto/components/ConfigureSTOForm/forms/USDTieredSTO/InvestmentTiers.js

```diff
@@ -247,6 +262,19 @@ class InvestmentTiers extends React.Component<Props, State> {
                             {row.cells.map(cell => (
                               <TableCell key={cell.id}>{cell.value}</TableCell>
                             ))}
+                            {row.id > 0 ? (
+                              <TableCell>
+                                <Icon
+                                  Icon={DeleteIcon}
+                                  color="#000000"
+                                  onClick={() => {
```

Comment:
Instead of doing this you should use the new `IconButton` component from `polymath-ui`. It renders a `<button />` component that is more semantic/accessible than using an `onClick` on a non-clickable element.

Reply:
I've added this change